src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubExtensionConfigProvider.cs

```diff
@@ -142,5 +156,25 @@ private static Task<object> ConvertPocoToEventData(object arg, Attribute attrRes
         {
             return Task.FromResult<object>(ConvertString2EventData(JsonConvert.SerializeObject(arg)));
         }
+
+        private sealed class EventHubClientBinder : IValueBinder
+        {
+            private readonly object _messageSender;
+            private readonly Type _type;
+
+            public EventHubClientBinder(object messageSender, Type type)
```

Comment:
Wondering why this is called "messageSender" when it is an EventClient? Why is the type you use object, and not EventClient?